servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -1050,6 +1066,25 @@ void RenderForwardMobile::_render_scene(RenderDataRD *p_render_data, const Color
 			breadcrumb = RDD::BreadcrumbMarker::REFLECTION_PROBES;
 		}
 
+		if (rb_data.is_valid()) {
+			RID mv_fb = rb_data->get_motion_vectors_fb();
+
+			if (mv_fb.is_valid()) {
+				RENDER_TIMESTAMP("Render Motion Vectors");
+
+				Vector<Color> mv_pass_clear;
+				mv_pass_clear.push_back(Color(0, 0, 0, 0));
+
+				RD::get_singleton()->draw_command_begin_label("Render Motion Vectors");
+
+				RID rp_uniform_set = _setup_render_pass_uniform_set(RENDER_LIST_OPAQUE, nullptr, RID(), samplers);
```

Comment:
I suspect that we will want to do something more like the Forward+ Renderer where we only render dynamic objects in this pass and then we run a quick full screen shader to calculate the Motion Vectors for static geometry from the depth buffer. 

For now this is fine since it will always be a low-res pass. But for non-spacewarp applications we will basically want to do the following in supasses:
1. Render static objects as normal
2. Render dynamic objects as normal + to the MV buffer
3. Do a full screen pass over the MV buffer and derive MVs for static objects from depth
4. Normal post effects pass. 

For phones, this should give us the best performance.